src/checkers/inference/solver/backend/encoder/SlotSlotCombo.java

```diff
@@ -3,9 +3,14 @@
 import checkers.inference.model.BinaryConstraint;
 import checkers.inference.model.CombineConstraint;
 import checkers.inference.model.Slot;
+import checkers.inference.solver.backend.encoder.binary.BinaryConstraintEncoder;
 
 /**
- * Created by mier on 06/11/17.
+ * Enum that models combination of slots so that encoders knows which encodeXXX() method should be
```

Comment:
"so that encoders knows which encodeXXX() method" -> "so that EncoderCoordinator knows which encodeXXX() method of encoders it should dispatch to".